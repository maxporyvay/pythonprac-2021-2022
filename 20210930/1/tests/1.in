1,5,8,2,4,9,125125,53156,548,236
