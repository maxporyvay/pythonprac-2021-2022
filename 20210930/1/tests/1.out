[1, 2, 548, 4, 5, 125125, 53156, 8, 9, 236]
